diesel_tests/build.rs

```diff
@@ -39,7 +39,7 @@ mod outer {
         let database_url = ::std::env::var("DATABASE_URL")
             .expect("DATABASE_URL must be set to run tests");
         let connection = PgConnection::establish(&database_url).unwrap();
-        let migrations_dir = migrations::find_migrations_directory().unwrap().join("postgresql");
+        let migrations_dir = migrations::find_migrations_directory().unwrap().join("../../migrations/postgresql");
```

Comment:
Ah, makes sense. Honestly, with this feature added, I'm thinking we might just be able to cut most of the existing "runtime" migration functionality, and move it into `diesel_cli`. Packaging the migrations in the binary feels more rust-like. That doesn't need to be part of this PR though, this is fine for now.